recipe/meta.yaml

```diff
@@ -24,13 +24,17 @@ requirements:
     - gcc  # [osx]
     - boost 1.63.*  # note this is intentionally NOT the latest version; synced with casacore which has special needs
     - casacore 2.2.*
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```

Comment:
> I'm still a bit confused about what requirements I could put instead

Sorry for that. 

> so I'll stick with the document ones for now.

I am updating the docs now though :smile: 

When I wrote that we did not have numpy 1.8 and 1.9 available for all pythons, that is why we ended up with a `numpy` for each python. The new docs will be:

```yaml
build:
    - numpy 1.8.*  # [not (win and py36)]
    - numpy 1.11.*  # [win and py36]
run:
    - numpy >=1.8  # [not (win and py36)]
    - numpy >=1.11  # [win and py36]
```

And, if you project allows, you can go as low as `numpy 1.7` for `Python 2.7`, but for simplicity I guess that 1.8 is good enough.